apps/hash-graph/bench/benches/read_scaling/knowledge/complete/entity.rs

```diff
@@ -106,8 +106,8 @@ async fn seed_db(
                             None,
```

Comment:
Just commenting to thread replies.

Do we know why the CI lint job is failing?

Reply:
There is a corresponding [Slack thread](https://hashintel.slack.com/archives/C02TWBTT3ED/p1675079929051119) (_internal_) and [a PR](https://github.com/hashintel/hash/pull/1973) to at least make the rest of the CI run.